.github/workflows/community.yml

```diff
@@ -11,5 +11,5 @@ jobs:
-          pr-message: '<h1>Welcome to CommunityPro</h1> <strong>Thank you for raising an Issue in this community!</strong> <p>If you are willing to join the community, connect via <ul><li><a href="https://chat.whatsapp.com/BMFVTksMg269GdgsviJzLm">WhatsApp</a></li><li><a href="https://www.linkedin.com/company/communitypro">Linkedin</a></li><li><a href="https://github.com/CommunityPro">Communitypro organization page</a> by raising an <a href="https://github.com/CommunityPro/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+Community+Organization">invitational issue</a></li></ul>We appreciate every contribution, so consider adding your profile in the <a href="https://github.com/CommunityPro/Practice-Contribution">pratice contribution</a> repo</p>'
+          issue-message: '<h1>Welcome to CommunityPro</h1> <strong>Thank you @contributor_name for raising an Issue in this community!</strong> <p>If you are willing to join the community, connect via <ul><li><a href="https://chat.whatsapp.com/BMFVTksMg269GdgsviJzLm">WhatsApp</a></li><li><a href="https://www.linkedin.com/company/communitypro">Linkedin</a></li><li><a href="https://github.com/CommunityPro">Communitypro organization page</a> by raising an <a href="https://github.com/CommunityPro/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+Community+Organization">invitational issue</a></li></ul>We appreciate every contribution, so consider adding your profile in the <a href="https://github.com/CommunityPro/Practice-Contribution">pratice contribution</a> repo</p>'
+          pr-message: '<h1>Welcome to CommunityPro</h1> <strong>Thank you @contributor_name for raising an Issue in this community!</strong> <p>If you are willing to join the community, connect via <ul><li><a href="https://chat.whatsapp.com/BMFVTksMg269GdgsviJzLm">WhatsApp</a></li><li><a href="https://www.linkedin.com/company/communitypro">Linkedin</a></li><li><a href="https://github.com/CommunityPro">Communitypro organization page</a> by raising an <a href="https://github.com/CommunityPro/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+Community+Organization">invitational issue</a></li></ul>We appreciate every contribution, so consider adding your profile in the <a href="https://github.com/CommunityPro/Practice-Contribution">pratice contribution</a> repo</p>'
```

Comment:
Awesome work @frankiefab100 but can you update the pr-message to:
Welcome to Communitypro. Thank you @contributor_name for raising a pull request. And then the other text will follow. 

Since it's a PR, we have to specify it accordingly. 🙏🏾 Is that okay?

Reply:
Sure, I will do that.

Reply:
How about modifying it to `... for making a pull request` instead of using `raising`